editor/src/core/workers/parser-printer/parser-printer.spec.ts

```diff
@@ -1907,44 +2010,14 @@ export var whatever = (props) => <View data-uid={'aaa'}>
     const exported = utopiaJSXComponent('whatever', true, defaultPropsParam, ['color'], view, null)
     const imports = addImport('cake', null, [importAlias('cake')], null, sampleImportsForTests)
     const expectedResult = clearParseResultUniqueIDs(
-      parseSuccess(imports, [exported], expect.objectContaining({}), null, null),
-    )
-    expect(actualResult).toEqual(expectedResult)
-  })
-  it('parses the code when it is a var', () => {
```

Comment:
What happened to this test?

Reply:
It's a duplicate:
https://github.com/concrete-utopia/utopia/blob/a9c7edc3ffe8b8a7e0b04d5a7dd6dea9d97ed1a9/editor/src/core/workers/parser-printer/parser-printer.spec.ts#L1914
https://github.com/concrete-utopia/utopia/blob/a9c7edc3ffe8b8a7e0b04d5a7dd6dea9d97ed1a9/editor/src/core/workers/parser-printer/parser-printer.spec.ts#L1951

Reply:
Ha, seems like that was my fault when removing old dead code!

Reply:
I totally went "Wait, I fixed this test..." when changing the test.